joint_trajectory_controller/test/test_trajectory_controller.cpp

```diff
@@ -176,6 +176,58 @@ TEST_P(TrajectoryControllerTestParameterized, check_interface_names_with_command
     command_interfaces.names, testing::UnorderedElementsAreArray(command_interface_names));
 }
 
+TEST_P(TrajectoryControllerTestParameterized, check_use_of_closed_loop_pid_adapter)
+{
+  SetUpTrajectoryController();
+
+  const auto state = traj_controller_->get_node()->configure();
+  ASSERT_EQ(state.id(), State::PRIMARY_STATE_INACTIVE);
+
+  EXPECT_FALSE(traj_controller_->get_disable_closed_loop_pid_adapter());  // default value
+
+  // closed loop PID adapter will be use when velocity- or effort-only interfaces are used
```

Comment:
```suggestion
  // closed loop PID adapter will be used when velocity- or effort-only interfaces are used
```

I think another way to say this is, "The joint trajectory controller always operates in the position domain. If velocity-only or effort-only interfaces are requested, a PID controller is used to convert position to velocity or effort."